public/src/main/java/com/revenuecat/purchases/models/StoreProduct.kt

```diff
@@ -10,30 +11,40 @@ import org.json.JSONObject
 /**
  * Represents an in-app product's or subscription's listing details.
  */
-
-abstract class StoreProduct:Parcelable {
+@Parcelize
```

Comment:
idea here is that we could have a single StoreProduct interface for BC4 and BC5, and kinda squeeze BC4 into pricing phases... i loved that idea of yours

there are definitely some open questions as to how that would work fully and whether we have certain fields be BC5/4-only

Reply:
Adding extra fields like pricingPhases and tags can be be applied to both and will be either empty or backfilled.
What about some BC5 specific implementation details like offerToken and ProductDetails? We can make them available although IMO it's a bit strange to include them in the public interface if the app developers shouldn't be using it, right?